src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -9,7 +9,15 @@
             [react-native.core :as rn]
```

Comment:
wow, what a huge one, is it possible to split?

Reply:
Will do, I think we can split, let's see how much though :)

Reply:
Done, moved buttons to a separate internal namespace, soundtrack to a separate component (which will be used in audio message componend by the way) and import them from this namespace.